src/components/comparison/ComparisonPanel.js

```diff
@@ -74,7 +74,7 @@ const render = (product, key) => {
     case 'applicationUri':
       return !!product[key] && <a href={product[key]} target='_blank' rel='noopener noreferrer'>Apply here</a>
     case 'additionalInformation':
-      return <AdditionalInfo additionalInfo={product[key]} tableCell/>
+      return !!product[key] ? <AdditionalInfo additionalInfo={product[key]} tableCell/> : ''
```

Comment:
Good catch! No need to return the empty string though. JSX uses returned `false` as an indication to not render anything. Like this: `return !!product[key] && <AdditionalInfo additionalInfo={product[key]} tableCell/>`

Reply:
Cheers, and nice to not have to pass the empty string.

Reply:
Please make the change (since it's in your repository) and once the commit is added, I'll merge this PR.

Reply:
Ok, change is made.